README.md

```diff
@@ -14,7 +14,7 @@ You can find documentation in the doc directory. It's built with sphinx using
 autodoc, so you'll need to install server dependencies to generate it properly -
 see the 'dev environment' section. Docs can be built with:
 
-`sphinx-build -b html ./doc ./build`
```

Comment:
These changes were pulled into master already, you can rebase.

Reply:
I tried to do that already, but git just says `Current branch parse-replay-duration is up to date.`

I can even see that the commit hashes are the same from this branch to the master branch so it might just be a bug in github that it's still displaying them

Reply:
Ah, I made another update to the readme and now it's showing correctly